src/QMCDrivers/SimpleFixedNodeBranch.h

```diff
@@ -278,13 +277,13 @@ struct SimpleFixedNodeBranch : public QMCTraits
   inline void regressQMCCounter() { iParam[B_COUNTER]--; }
 
   /** get the EstimatorManager */
-  EstimatorManagerBase* getEstimatorManager() { return MyEstimator; }
+  EstimatorManagerBase* getEstimatorManager() { return MyEstimator.get(); }
```

Comment:
This is quite awful but acceptable since it is solely to support the legacy Driver's desire of a Estimator pointer.